Libraries/LibWeb/WebGL/WebGL2RenderingContextImpl.cpp

```diff
@@ -3194,6 +3194,18 @@ JS::Value WebGL2RenderingContextImpl::get_vertex_attrib(WebIDL::UnsignedLong ind
     }
 }
 
+WebIDL::LongLong WebGL2RenderingContextImpl::get_vertex_attrib_offset(WebIDL::UnsignedLong index, WebIDL::UnsignedLong pname)
+{
+    if (pname != GL_VERTEX_ATTRIB_ARRAY_POINTER) {
+        set_error(GL_INVALID_ENUM);
+        return -1;
```

Comment:
Should we return -1 here instead of 0? Where does it say that? Couldn't find it in the specs